frontend/src/components/AuthCheck.vue.js

```
@@ -1,24 +1,52 @@
+onMounted(async () => {
+    try {
+        // If middleware redirected you to OIDC already, this won’t run until after login
+        await store.loadBootstrap();
+    }
+    finally {
+        ready.value = true;
     }
 });
-debugger; /* PartiallyEnd: #3632/script.vue */
+const isAuth = isAuthenticated;
+debugger; /* PartiallyEnd: #3632/scriptSetup.vue */
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)